start_bundle_coupled.R

```diff
@@ -100,6 +100,7 @@ parallel <- ifelse("--parallel" %in% argv, TRUE, FALSE)
 errorsfound <- 0
 startedRuns <- 0
 waitingRuns <- 0
+deletedFolders <- 0
```

Comment:
to be more transparent, the start script tells users about folders that have been (in test mode: would be) deleted.